plugin/pkg/admission/podpreset/admission.go

```diff
@@ -38,6 +38,7 @@ import (
 
 const (
 	annotationPrefix = "podpreset.admission.kubernetes.io"
+	exclusionSuffix  = "PodPresetOptOut"
```

Comment:
that annotation prefix is used on a pod to say what pod preset mutated its spec.

i think your key would mean that the "opt-out" pod preset had actually mutated it.  we should have a different prefix.